### ISSUE TITLE
Mac - OSError: Could not load shared object file: libllvmlite.dylib

### ISSUE THREAD

**Issue by RAHEYO** [PrajwalVandana/maestro-cli#6]
Erorr occured when trying to run `maestro add -Y <link> <tag>`
```

Traceback (most recent call last):
  File "/Users/raheyo/opt/anaconda3/bin/maestro", line 5, in <module>
    from maestro import cli
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/maestro.py", line 55, in <module>
    import helpers
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/helpers.py", line 23, in <module>
    from numba import jit  # NOTE: I think I'm in love with this decorator
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/numba/__init__.py", line 19, in <module>
    from numba.core import config
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/numba/core/config.py", line 15, in <module>
    import llvmlite.binding as ll
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/__init__.py", line 4, in <module>
    from .dylib import *
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/dylib.py", line 3, in <module>
    from llvmlite.binding import ffi
  File "/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/ffi.py", line 195, in <module>
    raise OSError(msg)
**OSError: Could not load shared object file: libllvmlite.dylib**

Errors were: [OSError("dlopen(/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/libllvmlite.dylib, 0x0006): Library not loaded: @rpath/libLLVM-11.dylib\n 
Referenced from: <30EC6215-F7F9-31EB-861D-2FF27BD00307> /Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/libllvmlite.dylib\n 
Reason: tried: '/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/../../../../libLLVM-11.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/../../../../libLLVM-11.dylib' (no such file),
'/Users/raheyo/opt/anaconda3/lib/python3.9/lib-dynload/../../libLLVM-11.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/bin/../lib/libLLVM-11.dylib' (no such file), 
'/usr/local/lib/libLLVM-11.dylib' (no such file), 
'/usr/lib/libLLVM-11.dylib' (no such file, not in dyld cache)"), OSError("dlopen(libllvmlite.dylib, 0x0006): tried: 
'libllvmlite.dylib' (no such file), 
'/System/Volumes/Preboot/Cryptexes/OSlibllvmlite.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/lib/python3.9/lib-dynload/../../libllvmlite.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/bin/../lib/libllvmlite.dylib' (no such file), 
'/usr/lib/libllvmlite.dylib' (no such file, not in dyld cache), 'libllvmlite.dylib' (no such file),
'/usr/local/lib/libllvmlite.dylib' (no such file), '/usr/lib/libllvmlite.dylib' (no such file, not in dyld cache)"), 
OSError("dlopen(./libllvmlite.dylib, 0x0006): tried: './libllvmlite.dylib' (no such file), 
'/System/Volumes/Preboot/Cryptexes/OS./libllvmlite.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/lib/python3.9/lib-dynload/../.././libllvmlite.dylib' (no such file), '/Users/raheyo/opt/anaconda3/bin/../lib/./libllvmlite.dylib' (no such file), 
'/usr/lib/./libllvmlite.dylib' (no such file, not in dyld cache), 
'./libllvmlite.dylib' (no such file), '/Users/raheyo/libllvmlite.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/raheyo/libllvmlite.dylib' (no such file), 
'/Users/raheyo/libllvmlite.dylib' (no such file)"), 
OSError("dlopen(/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/libllvmlite.dylib, 0x0006): Library not loaded: @rpath/libLLVM-11.dylib\n  
Referenced from: <30EC6215-F7F9-31EB-861D-2FF27BD00307> /Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/libllvmlite.dylib\n  
Reason: tried: '/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/../../../../libLLVM-11.dylib' (no such file), 
'/Users/raheyo/opt/anaconda3/lib/python3.9/site-packages/llvmlite/binding/../../../../libLLVM-11.dylib' (no such file), '/Users/raheyo/opt/anaconda3/lib/python3.9/lib-dynload/../../libLLVM-11.dylib' (no such file), '/Users/raheyo/opt/anaconda3/bin/../lib/libLLVM-11.dylib' (no such file), '/usr/local/lib/libLLVM-11.dylib' (no such file), '/usr/lib/libLLVM-11.dylib' (no such file, not in dyld cache)")]
```

The installation process went fine...

**Comment by PrajwalVandana** [comment]
Check if 1.0.10 fixes it

**Comment by RAHEYO** [comment]
Run `pip install numba --force-reinstall --ignore-installed llvmlite` fixed the issue instantly and 1.0.10 removes other errors as well. Now it works perfectly!
